currency/pair_methods.go

```diff
@@ -144,3 +147,77 @@ func (p Pair) Other(c Code) (Code, error) {
 func (p Pair) IsPopulated() bool {
 	return !p.Base.IsEmpty() && !p.Quote.IsEmpty()
 }
+
+// MarketSellOrderAspect returns an order aspect for when you want to sell a
+// currency which purchases another currency. This specifically returns what
+// liquidity side you will be affecting, what order side you will be placing and
+// what currency you will be purchasing.
+func (p Pair) MarketSellOrderAspect(wantingToSell Code) (*OrderAspect, error) {
+	return p.getAspect(wantingToSell, true, true)
+}
+
+// MarketBuyOrderAspect returns the order aspect for when you want to buy a
+// currency which sells another currency. This specifically returns what
+// liquidity side you will be affecting, what order side you will be placing and
+// what currency you will be selling.
+func (p Pair) MarketBuyOrderAspect(wantingToBuy Code) (*OrderAspect, error) {
+	return p.getAspect(wantingToBuy, false, true)
+}
+
+// LimitSellOrderAspect returns the order aspect for when you want to sell a
+// currency which purchases another currency. This specifically returns what
+// liquidity side you will be affecting, what order side you will be placing and
+// what currency you will be purchasing.
+func (p Pair) LimitSellOrderAspect(wantingToSell Code) (*OrderAspect, error) {
+	return p.getAspect(wantingToSell, true, false)
+}
+
+// LimitBuyOrderAspect returns the order aspect for when you want to buy a
+// currency which sells another currency. This specifically returns what
+// liquidity side you will be affecting, what order side you will be placing and
+// what currency you will be selling.
+func (p Pair) LimitBuyOrderAspect(wantingToBuy Code) (*OrderAspect, error) {
+	return p.getAspect(wantingToBuy, false, false)
+}
+
+// getAspect returns the order aspect for the currency pair using the provided
+// currency code, whether or not you are selling and whether or not you are
+// placing a market order.
+func (p Pair) getAspect(c Code, selling, market bool) (*OrderAspect, error) {
```

Comment:
While `aspect` doesn't seem right, I don't have any good descriptors without becoming verbose `getOrderSideDecisionDetailsForOrderIWishToMake(...)` especially since order.Side is its own concept

`OrderPair`? Yes it has `Pair` in the name, but you are re-arranging the pair details depending on how you wish to make the order

Reply:
I'm a fan of `getOrderDetails` or `getOrderParameters` as aspect is too broad

Reply:
I changed it to `OrderDecisionDetails` type